MapboxSearch.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MapboxSearch'
-  s.version          = '1.0.0-rc.2'
+  s.version          = '1.0.0-rc.3'
```

Comment:
Let's increase version for the `SearchSDKUI` podspec as well. Usually, we bump both libraries.
Here is an [example]( https://github.com/mapbox/mapbox-search-ios/pull/103/files#diff-50662b0f63d56eca9ff6b9c56939241dae4c8c679c41c7edab670be8d6424ad7).